library/oneview_alert_facts.py

```diff
@@ -0,0 +1,126 @@
+#!/usr/bin/python
+
+###
+# Copyright (2016) Hewlett Packard Enterprise Development LP
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# You may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+###
+
+from ansible.module_utils.basic import *
+try:
+    from hpOneView.oneview_client import OneViewClient
+
+    HAS_HPE_ONEVIEW = True
+except ImportError:
+    HAS_HPE_ONEVIEW = False
+
+DOCUMENTATION = '''
+module: oneview_alert_facts
+short_description: Retrieve facts about the OneView Tasks.
+description:
+    - Retrieve facts about the OneView Alerts.
+requirements:
+    - "python >= 2.7.9"
+    - "hpOneView >= 2.0.1"
```

Comment:
The version 2.0.1 from the OneView Python SDK was already released.
I suggest change to the next release, 3.0.0 (comming soon).